LondonBritishLibrary/orient/BLorient741.xml

```diff
@@ -53,49 +233,296 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                            <desc>Small ornamental band in red and black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d15" type="ornamentation">
+                            <locus target="#38va"></locus>
+                            <desc>Small ornamental band in black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d16" type="ornamentation">
+                            <locus target="#47rc"></locus>
+                            <desc>Small ornamental band in red and black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d17" type="ornamentation">
+                            <locus target="#49ra"></locus>
+                            <desc>Small ornamental band in red and black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d18" type="ornamentation">
+                            <locus target="#67vc"></locus>
+                            <desc>Small ornamental band in red and black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d19" type="ornamentation">
+                            <locus target="#69va"></locus>
+                            <desc>Small ornamental band in black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d20" type="ornamentation">
+                            <locus target="#74rb"></locus>
+                            <desc>Small ornamental band in red and black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d21" type="ornamentation">
+                            <locus target="#76va"></locus>
+                            <desc>Small ornamental band in black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d22" type="ornamentation">
+                            <locus target="#76va"></locus>
+                            <desc>Small ornamental band in black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d23" type="ornamentation">
+                            <locus target="#96ra"></locus>
+                            <desc>Small ornamental band in red and black.</desc>
+                          </decoNote>
+                          <decoNote xml:id="d24" type="ornamentation">
+                            <locus target="#100rc"></locus>
+                            <desc>Small ornamental band in black.</desc>
+                          </decoNote>
+                        </decoDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1v"></locus>
+                              <desc type="PurchaseNote">A note in Amharic, possibly a purchase note,
```

Comment:
This and a2 are two separate notes?

Reply:
Yes, I think they have been written by two different hands.